assets/js/index.js

```diff
@@ -0,0 +1,92 @@
+// TODO: Include packages needed for this application
+const inquirer = require("inquirer");
+const fs = require("fs");
+const generateMarkdown = require("../utils/generateMarkdown");
+
+// TODO: Create an array of questions for user input:
```

Comment:
remove the TODO comments please - these should only be present whilst you are working on your assignment and need to keep track of things to complete

Reply:
Done